engo_glfw.go

```diff
@@ -317,7 +352,8 @@ func SetVSync(enabled bool) {
 	}
 }
 
-func init() {
```

Comment:
See my response above, and I don't see how you want `flexibility` yet still removed the `Preload` method from the Scene interface.